src/client.ts

```diff
@@ -140,7 +140,7 @@ export default class Client implements Erdstall {
 
 		const exittx = new ExitRequest(this.address, await this.nextNonce());
 		await exittx.sign(this.erdstallConn.erdstall(), this.signer);
-		return this.enclaveConn.exit(exittx);
+		return await this.enclaveConn.exit(exittx);
```

Comment:
Any special reason for `await`ing every Promise?

Less performant in production vs just returning, but makes debugging a little bit easier AFAIK. Just curious.

Reply:
Just wanted to make sure that `await client.X()` only really returns when `X` finishes, and not before that (ignoring the special case of on-chain calls).